src/app/directives/course-card/course-card-style-creation.directive.spec.ts

```diff
@@ -0,0 +1,9 @@
+// import { CourseCardCreationDateDirective } from './course-card-creation-date.directive';
```

Comment:
Do we need this commented file?

Reply:
It is just for preventing warning at IDE that 'There are unused imports'